src/strings/pdf.js

```diff
@@ -53,36 +53,46 @@ const PDF_STRINGS = {
   parentalCustody: 'Parental Custody (check the appropriate custody status)',
   bothLiving:
     'Both parents are living, have legal custody of the minor child and have signed this document; ',
-  or: 'OR',
+  or: ' OR',
```

Comment:
This can be tricky with translations. I don't think we should rely on leading or trailing spaces in strings.

Reply:
OK. Is the better option ( space: ' ' ) inserted where necessary to break things up for formatting or to unify strings that require spacing?

Reply:
To be removed in the next PR.